lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  afterEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { mountWithContext } from '../../../../test/bigtest/helpers';
+import {
+  ProfilesLabelInteractor,
+  ProfilesPopoverInteractor,
+} from './interactors';
+import ProfilesLabel from '../ProfilesLabel';
+
+import translations from '../../../../translations/stripes-data-transfer-components/en';
+
+describe('ProfilesLabel', () => {
+  const profilesLabel = new ProfilesLabelInteractor();
+  const popover = new ProfilesPopoverInteractor();
+
+  describe('rendering ProfileLabel', () => {
+    const link = 'http://localhost/';
+    const content = <div data-profiles-label-content>Content</div>;
+
+    beforeEach(async () => {
+      const overlayContainer = document.createElement('div');
+
+      overlayContainer.id = 'OverlayContainer';
+      document.body.append(overlayContainer);
+
+      await mountWithContext(
+        <ProfilesLabel
+          link={link}
+          content={content}
+        />
+      );
+    });
+
+    afterEach(() => {
+      document.getElementById('OverlayContainer').remove();
+    });
+
+    it('should display ProfileLabel component', () => {
```

Comment:
I suggest to not bound to component names as they may change over time.
```suggestion
    it('should display profile label', () => {
```